docs/getting-started/Configuration.md

```diff
@@ -0,0 +1,72 @@
+# How to configure warden #
+
+Configuration options for warden can be specified by providing a configuration
+file on the command-line.
+
+## Command-line Options ##
+
+The options below are specified on the command-line.
+
+* `--config` - A configuration file to load. For more information on the format
+  of this file, see the **Configuration Files** section. Only one configuration
+  file can be specified. Multiple uses of this argument will result in only the
+  last configuration file being read.
+
+## Configuration Files ##
+
+Configuration files are JSON formatted. They are a single JSON object
+containing configuration values.
+
+### Default Configuration File ###
+
+The configuration file, located in warden/config/defaults.json below is the default settings for warden.
+
+~~~json
+{
+  "vault": {
+    "port": 8200,
+    "hostname": "localhost",
+    "token": "Place-Vault-Token-Here"
+  },
+  "service": {
+    "port": 8080,
+    "hostname": "localhost"
+  },
+  "log": {
+    "level": "info"
+  }
+}
+~~~
+
+### Configuration Key Reference ###
+
+* `vault` - These settings contain how warden will reach the vault server.
```

Comment:
Do we want to capitalize "vault" (not the key in the settings, but the "vault server" bit in the docs here)? It's capitalized in [the Vault documentation](https://www.vaultproject.io/docs/index.html).